dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/registry/ZoneAwareClusterInvoker.java

```diff
@@ -216,23 +209,16 @@ public Result doInvoke(Invocation invocation, final List<Invoker<T>> invokers, L
 
     private boolean shouldMigrate(boolean addressChanged, List<Invoker<T>>  serviceInvokers, List<Invoker<T>>  interfaceInvokers) {
```

Comment:
The indent can be format followed the checkstyle

Reply:
fixed it.